tests/unit/test_streamlit.py

```
@@ -44,3 +44,46 @@ def test_fetch_streamlit_apps(self):
                     }
                 ],
             )
+
+
+class TestStartStreamlitServers(unittest.TestCase):
+    def _make_apps(self, *entrypoints):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_make_apps`

(ANN202)